### ISSUE TITLE
WebMessageBodyStyle.Bare is not properly parsed to swagger.json

### ISSUE THREAD

**Issue by colbroth** [abelsilva/swaggerwcf#95]
When using WebMessageBodyStyle.Bare, I am getting errors in http://editor.swagger.io/ and I am unable to use swagger code-gen to generate client code. If I use WebMessageBodyStyle.Wrapped, it is properly parsed. For our framework, I cannot change our code to use WebMessageBodyStyle.Wrapped, so I am hoping you can guide me to a fix.

For example, I added to the SwaggerWcf example file IStore.cs, the following API.

 [WebInvoke(UriTemplate = "/booksint", BodyStyle = WebMessageBodyStyle.Bare, //Method = "POST",
 RequestFormat = WebMessageFormat.Json, ResponseFormat = WebMessageFormat.Json)]
 [OperationContract]
 bool CreateBookInt(int value);

The corresponding Store.svc.cs simpley has

        public bool CreateBookInt(int value)
        {
            return true;
        }

The swagger.json file converted to yaml using http://editor.swagger.io/ produces the following output:

/v1/rest/booksint:
    post:
      summary: Create book
      description: Create a book on the store
      operationId: SwaggerWcf.Test.Service.BookStore.CreateBookInt
      consumes:
        - application/json
      produces:
        - application/json
      parameters:
        - name: value
          in: body
          description: 'Book to be created, the id will be replaced'
          required: true
          **type: integer
          format: int32**
      responses:
        default:
          description: not available
          schema:
            type: boolean

The editor.swagger.io indicates error because type and format are not allowed for PUT parameter.
"Schema error at paths['/v1/rest/booksint'].post.parameters[0]
should NOT have additional properties
additionalProperty: type, format, name, in, description, required"

When I try to use swagger code-gen, a null exception is throw because of this.

If I use WebMessageBodyStyle.Wrapped, the yaml has proper syntax as follows:

/v1/rest/booksint:
    post:
      summary: Create book
      description: Create a book on the store
      operationId: SwaggerWcf.Test.Service.BookStore.CreateBookInt
      consumes:
        - application/json
      produces:
        - application/json
      parameters:
        - name: CreateBookIntRequestWrapper
          in: body
          required: true
          **schema:
            $ref: '#/definitions/CreateBookIntRequest'**
      responses:
        default:
          description: not available
          schema:
            $ref: '#/definitions/CreateBookIntResultWrapper'

**Comment by abelsilva** [comment]
can you please recheck?

I've tested and it's working...
No warnings/errors

Interface:
```

        [SwaggerWcfPath("Test", "Test")]
        [WebInvoke(UriTemplate = "/books/{id}/test", BodyStyle = WebMessageBodyStyle.Bare,
            RequestFormat = WebMessageFormat.Json,
            ResponseFormat = WebMessageFormat.Json)]
        [OperationContract]
        bool Test(int id);
```

Class:
```

        [SwaggerWcfTag("Books")]
        [SwaggerWcfResponse(HttpStatusCode.OK, "Book found, author value in the response body")]
        public bool Test(int id)
        {
            return 0;
        }
```

![image](https://user-images.githubusercontent.com/1167088/30722583-711a1aba-9f08-11e7-885f-827a6dea989e.png)

**Comment by colbroth** [comment]
Yes your example works. However, the "id" is not in the URL for the failure I am seeing. The "id" is in the body. Could you please try:

Interface:


```
        [SwaggerWcfPath("Test", "Test")]
        [WebInvoke(UriTemplate = "/books/test", BodyStyle = WebMessageBodyStyle.Bare,
            RequestFormat = WebMessageFormat.Json,
            ResponseFormat = WebMessageFormat.Json)]
        [OperationContract]
        bool Test(int id);
```

**Comment by colbroth** [comment]
I have modified your example of what I expect the swagger.json file to output when using WebMessageBodyStyle .Bare and Request body has an integer. The file is attached.

[swaggerBook.txt](https://github.com/abelsilva/swaggerwcf/files/1338434/swaggerBook.txt)

Basically I changed: 

`
"/v1/rest/books/test": {
			"post": {
"parameters": [{
    "name": "id",
    "in": "body",
    "required": true,
    **"type": "integer",
    "format": "int32"**
  }
],`

To 

`
"/v1/rest/books/test": {
			"post": {
"parameters": [{
    "name": "id",
    "in": "body",
    "required": true,
     **"schema": {
          "type": "integer"
     }**
  }
],`

**Comment by abelsilva** [comment]
can you check if it is fixed now? sorry for the long delay

**Comment by colbroth** [comment]
This is now fixed in release 0.2.1. Thank you
